### PR TITLE
Add admin form

### PR DESCRIPTION
**Pull request by RedPandore**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -18,4 +26,41 @@ public function index(): Response
     {
         return $this->render('admin/index.html.twig');
     }
+
+    /**
+     * @Route("/add", name="add")
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php]: route should be in french and be more self-explanatory (what are we adding here?)

File: src/Controller/AdminController.php [hunk 2]
```diff
@@ -18,4 +26,41 @@ public function index(): Response
...
+
+            $admin->setCreatedAt(new DateTime('now'));
+            $admin->setRoles('ROLE_ADMIN');
+            $entityManager = $this->getDoctrine()->getManager();
+            $entityManager->persist($admin);
+            $entityManager->flush();
+
+            $this->addFlash(
+                'success',
+                'L\'inscription s\'est bien deroulée, un mail de confirmation a était envoyé à l\'administrateur'
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php]: but it doesn't look like a mail has been sent?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by bouboumael — DISMISSED** [review]

File: src/Controller/AdminController.php [hunk 3]
```diff
@@ -18,4 +26,42 @@ public function index(): Response
     {
         return $this->render('admin/index.html.twig');
     }
+
+    /**
+     * @Route("/creation_compte", name="creation_compte")
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php]: route should be in french, but route name should be in english

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by bouboumael — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**